src/BaseExportAdapter.php

```diff
@@ -76,11 +77,12 @@ protected function handleDbError(Throwable $e, int $maxRetries, ?string $outputT
         $message = $outputTable ? sprintf('[%s]: ', $outputTable) : '';
         $message .= sprintf('DB query failed: %s', $e->getMessage());
 
-        // Retry mechanism can be disabled
-        if ($maxRetries > 1) {
+        // Retry mechanism can be disabled or modified, ... we need to print real count
+        if ($e instanceof UserRetriedException && $e->getTryCount() > 1) {
             $message .= sprintf(' Tried %d times.', $maxRetries);
```

Comment:
- I want to disable retry mechanism in ex-hive for `SemanticException`.
- But I need to fix this bad message first.
- This message print the number of retries from the configuration, regardless of whether the retry have actually been performed so many times (it works well for the default `SimpleRetryPolicy` implementation, but not for the custom one).
- Therefore, the number of retries is extracted from the exception to which I added this value.